recipes/tachyon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.99b6" %}
+{% set name = "tachyon" %}
+{% set md5 = "9e5ed4045c472e01e57480d7c7eee849" %}
```

Comment:
Yeah `md5` is easily broken at this point. So we really should not use it if at all possible.